dojo/fixtures/dojo_testdata.json

```diff
@@ -1686,6 +1686,24 @@
       "epic_name_id": 333
     }
   },
+  {
+    "pk": 5,
+    "model": "dojo.jira_instance",
+    "fields": {
+      "username": "defect.dojo@testme.com",
+      "default_issue_type": "Bug",
+      "finding_text": "",
+      "low_mapping_severity": "test severity",
+      "url": "http://www.testjira.com",
+      "medium_mapping_severity": "test severity",
+      "open_status_key": 333,
+      "close_status_key": 334,
+      "critical_mapping_severity": "test severity",
+      "high_mapping_severity": "test severity",
+      "password": "user69",
+      "epic_name_id": 666
+    }
+  },
```

Comment:
do we need this?

Reply:
ah yes to make one with an email address.

Reply:
yea, just have to fiddle with extra things now in the fixture...

Reply:
easier just to change the username on jira_instance with id 2 in python code and do .save()

Reply:
fixtures are a pain and should onyl be used for a very minimal set of data to get the instance going.